python/vowpalwabbit/pyvw.py

```diff
@@ -1080,18 +1091,22 @@ def __str__(self):
         return s
 
 
-class multiclass_probabilities_label(abstract_label):
+class MulticlassProbabilitiesLabel(AbstractLabel):
     """Class for multiclass VW label with probabilities"""
 
-    def __init__(self, label, prediction=None):
-        abstract_label.__init__(self)
+    def __init__(
+        self,
+        label: Optional[pylibvw.example] = None,
+        prediction: Optional[float] = None,
+    ):
+        AbstractLabel.__init__(self)
         if isinstance(label, example):
             self.from_example(label)
         else:
             self.prediction = prediction
 
-    def from_example(self, ex):
-        self.prediction = get_prediction(ex, PredictionType.MULTICLASSPROBS)
+    def from_example(self, ex: pylibvw.example):
+        self.prediction = get_prediction(ex, pylibvw.vw.pMULTICLASSPROBS)
```

Comment:
Why remove the enum?

```suggestion
        self.prediction = get_prediction(ex, PredictionType.MULTICLASSPROBS)
```

Reply:
whoops typo from merging